### PR TITLE
migrations

### PR DESCRIPTION
**Pull request by Nderkovich**

### CODE REVIEW

File: drf_store/store/migrations/0001_initial.py [hunk 1]
```diff
@@ -54,6 +55,7 @@ class Migration(migrations.Migration):
                 ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.Group', verbose_name='groups')),
                 ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
                 ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.Permission', verbose_name='user permissions')),
+                ('email', models.EmailField(default=1983, max_length=70, unique=True)),
```
[the remark this answers is not in the document]
Comment (Nderkovich) [reply drf_store/store/migrations/0001_initial.py]: Removed.

File: drf_store/store/migrations/0001_initial.py [hunk 2]
```diff
@@ -68,11 +70,15 @@ class Migration(migrations.Migration):
...
-            name='email',
-            field=models.EmailField(default=1983, max_length=70, unique=True),
-            preserve_default=False,
+            name='first_name',
+            field=models.CharField(blank=True, max_length=50),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='last_name',
+            field=models.CharField(blank=True, max_length=50),
```
Comment (aleksey-kuchik) [inline drf_store/store/migrations/0001_initial.py:82]: If you're modifying the initial migration, why user `AlterField` when you can just change these fields in their definitions above?

Comment (Nderkovich) [reply drf_store/store/migrations/0001_initial.py:82]: Thought I should leave it because it was created by squashmigrations.

Comment (aleksey-kuchik) [reply drf_store/store/migrations/0001_initial.py:82]: `squashmigrations` is not the right tool to use in our case. It squashes the migrations and creates a new one. It is intended to be used when we have lots of migrations already applied and when we want to replace them with one migration. But in our case the migrations were not applied yet in previous PR. You should have just removed all of those migrations and run `makemigrations` again to get fresh migration with just create statements. Let's assume that migrations were not run in prod and use this approach.

Comment (Nderkovich) [reply drf_store/store/migrations/0001_initial.py:82]: Is it okay that it still creates `AddField` options for ManyToMany fields if the models they references were created after the main model?

Comment (aleksey-kuchik) [reply drf_store/store/migrations/0001_initial.py:82]: Yes, that's fine, this is how many to many works - you need both tables created before you can actually tie them with this relationship.

Comment (Nderkovich) [reply drf_store/store/migrations/0001_initial.py]: Okay, replaced migration with the one created by `makemigrations`.

**Review by aleksey-kuchik — APPROVED** [review]

**Pull request merged**